Dockerfile

```diff
@@ -26,6 +26,7 @@ EXPOSE 8000
 ENV LONGHORN_MANAGER_IP http://localhost:9500
 ENV LONGHORN_UI_PORT 8000
 
-USER nginx
+RUN usermod -u 499 nginx
```

Comment:
0-999 is usually for system accounts. Suggest changing to 1000.

Reply:
There is something weird that only using UID 499 would not have the `Permission denied` problem.
```
nginx: [alert] could not open error log file: open() "/var/log/nginx/error.log" failed (13: Permission denied)
2022/07/26 04:54:46 [warn] 10#10: duplicate MIME type "text/html" in /var/config/nginx/nginx.conf:7
2022/07/26 04:54:46 [emerg] 10#10: open() "/var/log/nginx/access.log" failed (13: Permission denied)
```

Reply:
OK, changed UID to 1000 and re-change the owner /var/log/nginx directory.

Reply:
This should be because you changed the original uid of the Nginx user after the Nginx has been installed and related artifacts (including some folders) are already authorized. 

Could you try to use the uid of the nginx user instead? No need to do any uid update actually. 
```
USER 499
```

Reply:
Yes, it works, but would it always be UID 499?

Reply:
Hmm because we relay on the package from BCI repo, this is not what we can change.

Reply:
Have u verified if after this change,  longhorn-hi will run well on a CIS hardened cluster?

Reply:
Yes, only update 'USER 499', it runs well on CIS hardened cluster I built.